packages/kit/src/runtime/client/router.js

```diff
@@ -250,8 +250,8 @@ export class Router {
 	 * @param {string} [hash]
 	 */
 	async _navigate(url, scroll, keepfocus, chain, hash) {
+		unselect_all(window);
```

Comment:
Assuming after we inline the code, I'm not sure if placing the unselect logic here is correct. I have a feeling it should be at [this line](https://github.com/sveltejs/kit/blob/93726c38d2118b1dc136f3ef659c5ae6bb039138/packages/kit/src/runtime/client/renderer.js#L277), but @benmccann may know better where this fits.